simple/src/codingPractice/SortArray_ReadMe

```diff
@@ -0,0 +1,6 @@
+Java Program to sort the elements of an array in ascending order
+"C:\Program Files\Java\jdk-17.0.4.1\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\lib\idea_rt.jar=61065:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\suyogw\Desktop\Auto\NewMyWayRepo\simple\out\production\simple codingPractice.SortArray
+Elements of original array:
+5 2 8 7 1  Elements of array sorted in ascending order:
```

Comment:
Output(Elements of array sorted in ascending order) should be in the next line.

Reply:
done